app/assets/v2/js/hackathon-projects.js

```diff
@@ -29,8 +29,12 @@ const submitProject = (logo, data, callback) => {
     if (callback) {
       callback(response);
     }
+    if (document.location.pathname.includes('fulfill')) {
+      document.location.reload();
```

Comment:
why do we reload ? i didn't get this 😅

Reply:
is not the most elegant way I know but the form need the project pk and that hackathon projects is used in many places, so in some point we can just add that to the data form, but that will need a lot of changes on the form approach, and as we are planning to move that form to vue doesn't make sense to invest a lot of time. fair?